README.md

```diff
@@ -1,67 +1,93 @@
 # conjur-api-python3
 
-Python3-based API SDK for [Conjur OSS](https://www.conjur.org/). The repo
-also includes a self-contained CLI tool (`conjur-cli`) that wraps the API
-in a simple executable script/binary.
+Command-line tool and programmatic Python access to the Conjur API to manage your Conjur resources. 
+
+This repository includes both the self-contained Conjur command-line tool (`conjur`) and Python3-based SDK for Conjur.
 
 [![Test Coverage](https://api.codeclimate.com/v1/badges/d90d69a3942120b36785/test_coverage)](https://codeclimate.com/github/cyberark/conjur-api-python3/test_coverage) [![Maintainability](https://api.codeclimate.com/v1/badges/d90d69a3942120b36785/maintainability)](https://codeclimate.com/github/cyberark/conjur-api-python3/maintainability)
 
 ---
 
-### **Status**: Alpha
+## Certificate level
 
-#### **Warning: Naming and APIs are still subject to breaking changes!**
+### Conjur CLI
 
----
+![](https://img.shields.io/badge/Certification%20Level-Certified-6C757D?link=https://github.com/cyberark/community/blob/master/Conjur/conventions/certification-levels.md)
+
+The Conjur CLI is a **Certified** level project. It's been reviewed by CyberArk to verify that it will securely
+work with CyberArk DAP as documented. In addition, CyberArk offers Enterprise-level support for these features. For
```

Comment:
not sure what you want to do about DAP vs CE at this point

Reply:
@InbalZilberman ?

Reply:
It will be a separate effort I believe. A separate effort dedicated to this name change